src/UI/Internal/Filter/View.elm

```diff
@@ -508,3 +452,299 @@ sortingIcon renderConfig attrs iconSize sorting =
+
+                    Model.MultiTextFilter { editable } ->
+                        defaultMultiTextFilter renderConfig size config.label editable
+
+                    Model.SelectFilter list { editable } ->
+                        selectFilterRender renderConfig size config.label editable list
+
+                    Model.SingleDateFilter { editable } ->
+                        singleDateFilterRender renderConfig size config.label editable
+
+                    Model.RangeDateFilter { editable } ->
+                        rangeDateFilterRender renderConfig size config.label editable
+
+                    Model.PeriodDateFilter { editable } ->
+                        periodDateFilterRender renderConfig size config.label editable
+    in
+    { label = config.label
+    , openMsg = config.openMsg
+    , closeMsg = config.closeMsg
+    , open = config.isOpen
+    , width = Element.fill
+    , rowsHeight = Just 160
+    , size = Medium
+    , sorting = Just sortingData
+    , buttons = defaultButtons config.editMsg filter
+    , applied =
+        Maybe.map
+            (\i -> { preview = String.fromInt i, clearMsg = config.editMsg Msg.Clear })
+            (Model.appliedLength filter)
+    , rows = rows
+    }
+
+
+defaultSingleTextFilter :
+    RenderConfig
+    -> FilterSize
+    -> String
+    -> Model.Editable String
+    -> List (Element Msg)
+defaultSingleTextFilter renderConfig size label editable =
+    editable
+        |> Model.editableWithDefault ""
+        |> TextField.singlelineText Msg.EditSingleText label
+        |> TextField.withSize (sizeToElement size)
+        |> TextField.withWidth TextField.widthFull
+        |> TextField.withOnEnterPressed Msg.Apply
+        |> TextField.renderElement renderConfig
+        |> List.singleton
+
+
+defaultMultiTextFilter :
+    RenderConfig
+    -> FilterSize
+    -> String
+    -> Model.Editable (Array String)
+    -> List (Element Msg)
+defaultMultiTextFilter renderConfig size label editableArr =
+    let
+        rowField subIndex line =
+            line
+                |> TextField.singlelineText (Msg.EditMultiText subIndex) label
+                |> TextField.withSize (sizeToElement size)
+                |> TextField.withWidth TextField.widthFull
+                |> TextField.withOnEnterPressed Msg.Apply
+                |> TextField.renderElement renderConfig
+    in
+    editableArr
+        |> Model.editableWithDefault Array.empty
+        |> Array.push ""
+        |> Array.indexedMap rowField
+        |> Array.toList
+
+
+selectFilterRender :
+    RenderConfig
+    -> FilterSize
+    -> String
+    -> Model.Editable Int
+    -> List String
+    -> List (Element Msg)
+selectFilterRender renderConfig size _ { current, applied } list =
+    Radio.group
+        { label = renderConfig |> RenderConfig.localeTerms >> .filters >> .select >> .description
+        , onSelectMsg = always Msg.EditSelect
```

Comment:
can you also handle the index and focus on the elements?

Reply:
I just can't do anything about tables, there is no way to do an `Effect Msg -> Effect msg` in their current API...